staging/src/kubevirt.io/api/export/v1alpha1/types.go

```diff
@@ -80,6 +80,11 @@ type VirtualMachineExportStatus struct {
 	// +optional
 	Links *VirtualMachineExportLinks `json:"links,omitempty"`
 
+	// +optional
+	// DefaultSecretExportName is the name of an optional secret specifically created for the current VirtualMachineExport object
+	// in case no other secret name is specified when creating the object
+	DefaultSecretExportName string `json:"defaultSecretExportName,omitempty"`
```

Comment:
I would just call this `TokenSecretRef` and set it to whatever the user set explicitly or the one we create.  One source of truth to get the secret name